src/rpm_ostree/mod.rs

```diff
@@ -17,6 +21,35 @@ pub struct Release {
     pub version: String,
     /// Image base checksum.
     pub checksum: String,
+    /// Release age (Cincinnati `age_index`).
+    pub age_index: Option<u64>,
+}
+
+impl std::cmp::Ord for Release {
+    fn cmp(&self, other: &Self) -> Ordering {
+        let self_age = self.age_index.clone().unwrap_or(0);
+        let other_age = other.age_index.clone().unwrap_or(0);
+
+        if self_age != other_age {
+            return self_age.cmp(&other_age);
+        }
+
+        if self.version != other.version {
+            return self.version.cmp(&other.version);
```

Comment:
OK, just to confirm: in theory, we shouldn't even get past this because the age index of all the nodes in the graph should be different, right?

This is probably worth a comment here that comparing version strings and checksums are just to get a stable order in that event, even though it's not necessarily a meaningful order at the high-level.

Reply:
Correct!
I've added a comment to make that explicit.